src/components/videocard/videocard.css

```diff
@@ -0,0 +1,153 @@
+.grand-video-card {
+    font-size: 1.3rem;
+    position: relative;
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start;
+    align-items: stretch;
+    max-width: 30rem;
```

Comment:
you can use percentage also to give width value

Reply:
That would actually cause issue while making screen responsive so  kept it that way